app/scripts/modules/openstack/instance/osInstanceTypeSelectField.directive.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.openstack.instance', [
```

Comment:
Might want to make the package more specific to the component, e.g. `spinnaker.openstack.instance.instanceTypeSelectField`